squidb/src/com/yahoo/squidb/utility/VersionCode.java

```diff
@@ -18,6 +18,13 @@
     public static final VersionCode V3_7_4 = new VersionCode(3, 7, 4, 0); // Default on API 14, default minimum
```

Comment:
Good question. We had defined them because they were sort of "significant" SQLite versions, the former being the min version of SQLite found in Android API level 14 and the latter being when common table expressions were introduced (which I was looking at implementing at one point but decided against). I bet even `V3_7_11` is really only needed for things like multi-row insert. Would it be cleaner to not have any of these constants declared here at all, and just allow each separate use case to define them as necessary?